src/containers/PagingInfo.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+
+import withAppSearch from "../app-search/withAppSearch";
+import PagingInfo from "../components/PagingInfo";
+
+class PagingInfoContainer extends Component {
+  static propTypes = {
+    current: PropTypes.number.isRequired,
+    results: PropTypes.arrayOf(PropTypes.object).isRequired,
+    resultsPerPage: PropTypes.number.isRequired,
+    searchTerm: PropTypes.string.isRequired,
+    totalResults: PropTypes.number.isRequired
+  };
+
+  render() {
+    const {
+      current,
+      results,
+      resultsPerPage,
+      searchTerm,
+      totalResults
+    } = this.props;
+    const start = totalResults === 0 ? 0 : (current - 1) * resultsPerPage + 1;
+    const end =
+      totalResults <= resultsPerPage
+        ? totalResults
+        : start + resultsPerPage - 1;
+
```

Comment:
I've always heard to have the operators at the end of the line. What about this?
```
    const end = totalResults <= resultsPerPage ? 
        totalResults :
        start + resultsPerPage - 1;
```

Reply:
That may have been autoformatted, I'll have to check.

Reply:
It was autoformatted that way, plus, I think I prefer that anyway.